### PR TITLE
add implement some functions such as getset strlen exist persist expire append in cli string(#188)

### PR DESCRIPTION
**Pull request by wangchenguang123**
6fc07f9bcb2fe37bf63ade271a83fa0
6945427a16cdfce3af05359015c2919

### CODE REVIEW

**Comment by sjcsjc123** [comment]
can show me when expire a key, can i get the key again?

**Comment by wangchenguang123** [comment]
> 

Oh, this is my mistake. When the key expires, I should further handle it. May I ask if it is a deletion or other operation

**Comment by wangchenguang123** [comment]
> can show me when expire a key, can i get the key again?

Oh, this is my mistake. When the key expires, I should further handle it. May I ask if it is a deletion or other operation

**Comment by sjcsjc123** [comment]
if expire, when get the key, it should create error to tell user the key is not existed

**Comment by wangchenguang123** [comment]
> if expire, when get the key, it should create error to tell user the key is not existed

ok，i know it

File: cmd/client/root.go [hunk 1]
```diff
@@ -31,4 +32,59 @@ func register(app *grumble.App) {
...
+		Help: "str value length",
+		Run:  stringStrLen,
+		Args: func(a *grumble.Args) {
+			a.String("key", "key", grumble.Default(""))
+		},
+	})
+
+	app.AddCommand(&grumble.Command{
+		Name: "getset",
+		Help: "get curvalue set newvalue",
```
Comment (qishenonly) [inline cmd/client/root.go:46]: The content of `Help` needs to be modified to indicate what data structure this is.  As shown in the figure below, you need to indicate which data structure to add or get data into.

The same below.

image

**Review by qishenonly — CHANGES_REQUESTED** [review]

**Pull request closed without merging**